src/App/screens/Instructor/components/InstructorNav.js

```diff
@@ -10,16 +10,8 @@ const InstructorNav = ({pathname}) => (
         route: '',
       },
       {
-        text: 'Topics',
-        route: '/topics',
-      },
-      {
-        text: 'Submit Lesson',
-        route: '/submit',
-      },
-      {
-        text: 'Lesson Submissions',
-        route: '/submissions',
+        text: 'Published Lessons',
+        route: '/published',
       },
```

Comment:
Moving the paginated list to an unfinished /published route. Will probably work on this next. Also removed routes that were just placeholders; I don't like unused code and will add them back when we get to them :)